cpp/include/raft/matrix/detail/linewise_op.cuh

```diff
@@ -484,6 +517,7 @@ void matrixLinewiseVecRows(Type* out,
     const uint expected_grid_size = rowLen / raft::gcd(block_work_size, uint(rowLen));
     // Minimum size of the grid to make the device well occupied
     const uint occupy = getOptimalGridSize<BlockSize>();
+    // todo(lsugy): alignedLen instead of totalLen?
```

Comment:
Good point! Do any tests fail if we change this to `alignedLen`? Could be a typo, but could be an important undocumented quirk as well :)

Reply:
All tests pass with the change.